src/cbc_sdk/audit_remediation/models/differential.yaml

```diff
@@ -0,0 +1,23 @@
+type: object
```

Comment:
This doesn't properly reflect the _info object of the Differential class instead this represents the query request json. This needs modified to represent singular result of the response